modules/rosa-cluster-autoscaler-cli-settings.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-cluster-autoscaling.adoc
+// * osd_cluster_admin/osd-cluster-autoscaling.adoc
+
+:_content-type: REFERENCE
+[id="rosa-cluster-cli-autoscale-settings_{context}"]
+= Cluster autoscaling settings using the ROSA CLI
+
```

Comment:
Are these settings, options, or parameters? Should probably use the same term throughout.

Reply:
Do these parameters appear in the interactive cluster install?